gtk/gtk-play.c

```diff
@@ -1051,6 +1051,47 @@ create_tracks_menu (GtkPlay * play, GstPlayerMediaInfo * media_info, GType type)
   return menu;
 }
 
+static gboolean
+player_set_rate_cb (GtkRange * range, GtkScrollType scroll,
+    gdouble value, GtkPlay * play)
+{
+  gst_player_set_rate (play->player, value);
+}
+
+static void
+player_speed_clicked_cb (GtkButton * button, GtkPlay * play)
+{
+  GtkWidget *scale;
+  GtkWidget *dialog;
+  GtkWidget *content;
+  GtkWidget *box;
+  gdouble value;
+
+  dialog = gtk_dialog_new ();
+  gtk_window_set_transient_for (GTK_WINDOW (dialog), GTK_WINDOW (play));
+  gtk_window_set_title (GTK_WINDOW (dialog), "Playback speed control");
+
+  content = gtk_dialog_get_content_area (GTK_DIALOG (dialog));
+  box = gtk_box_new (GTK_ORIENTATION_HORIZONTAL, 1);
+  gtk_box_pack_start (GTK_BOX (content), box, TRUE, TRUE, 0);
+
+  scale = gtk_scale_new_with_range (GTK_ORIENTATION_HORIZONTAL,
```

Comment:
i saw @amorenoc nice look GUI, maybe once that is completed then we can borrow the idea from it and then make similar looks and feel modification in gtk app...